From 219795dcc8b89c0161532bae0c00cf04b070ddd5 Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Sun, 22 Sep 2024 18:12:54 +0100
Subject: [PATCH 2/2] Compatibility for Python 3.13+

---
 ext/ae.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/ext/ae.c b/ext/ae.c
index 386c8cf..9f7199a 100644
--- a/ext/ae.c
+++ b/ext/ae.c
@@ -977,7 +977,7 @@ static OSErr GenericEventHandler(const AppleEvent *request, AppleEvent *reply, S
 		err = -1;
 		goto cleanup;
 	}
-	res = PyEval_CallObject(handler, args);
+	res = PyObject_CallObject(handler, args);
 	requestObject->ob_itself.descriptorType = 'null';
 	requestObject->ob_itself.dataHandle = NULL;
 	replyObject->ob_itself.descriptorType = 'null';
@@ -1013,7 +1013,7 @@ static OSErr GenericCoercionHandler(const AEDesc *fromDesc, DescType toType, SRe
 		err = -1;
 		goto cleanup;
 	}
-	res = PyEval_CallObject(handler, args);
+	res = PyObject_CallObject(handler, args);
 	fromObject->ob_itself.descriptorType = 'null';
 	fromObject->ob_itself.dataHandle = NULL;
 	Py_DECREF(args);
-- 
2.45.2

